src/test/java/htsjdk/samtools/CRAMComplianceTest.java

```diff
@@ -294,9 +293,6 @@ public void testBAMThroughCRAMRoundTrip() throws IOException, NoSuchAlgorithmExc
         // retrieve all records from the bam and reset the indexing bins to keep comparisons with
```

Comment:
update the comment?

Reply:
I've just removed the comment as all that line now does is load the records from a file and that didn't seem comment-worthy.